nrich-search-spring-boot-starter/README.md

```diff
@@ -4,11 +4,15 @@
 
 ## Overview
 
-Spring Boot starter for `nrich-search` module. The purpose of `nrich-search` is to provide easy querying of JPA entities through automatic query creation from passed parameters. It is built on top
-of `spring-data-jpa` module and adds custom repository interfaces whose implementation provides searching functionality. Searching is possible with a simple string and a list of searched properties or
-by using a separate class whose properties will be used to create a query. Additional configuration of generated query is possible with `SearchConfiguration` configuration class.
-Starter module provides a `@Configuration` class (`NrichSearchAutoConfiguration`) with default configuration of `nrich-search` module (while allowing for overriding with conditional annotations),
-and a `@ConfigurationProperties` class (`NrichSearchProperties`) with default configured values and does automatic registration through `spring.factories`.
+This module is a Spring Boot starter for the [nrich-search](../nrich-search/README.md) module.
+The purpose of `nrich-search` is to provide easy querying of JPA entities through automatic query creation from passed parameters.
+It is built on top of `spring-data-jpa` module and adds custom repository interfaces whose implementation provides searching functionality.
+Searching is possible with a simple string and a list of searched properties or by using a separate class whose properties will be used to create a query.
+Additional configuration of the generated query is possible with `SearchConfiguration` configuration class.
+
+Starter module provides a `@Configuration` class (`NrichSearchAutoConfiguration`) with default configuration of `nrich-search` module,
```

Comment:
Maybe add a link to the Java classes we are referring to?

Reply:
Good idea!

Reply:
We are still missing the link for `NrichSearchProperties`. Not sure about other Java classes in this README, should we add links to all?

Additionally, I think we can omit parentheses here.

Reply:
I was thinking about this and we can combine inline code block and a link (for example as this [`ResponseBodyAdvice`](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/servlet/mvc/method/annotation/ResponseBodyAdvice.html)) for every class. It is inconspicuous in rendered view, but in raw markdown it brings a lot of "noise" while reading so I didnt do it from the get go.

Reply:
I agree that it brings "noise" and I like the approach with reference links you introduced.